pkg/csi/service/vanilla/controller.go

```diff
@@ -175,9 +175,15 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 				}
 				log.Debugf("fsnotify event: %q", event.String())
 				if event.Op&fsnotify.Remove == fsnotify.Remove {
-					log.Infof("Reloading Configuration")
-					c.ReloadConfiguration(ctx)
-					log.Infof("Successfully reloaded configuration from: %q", cfgPath)
+					for {
+						reloadConfigErr := c.ReloadConfiguration()
+						if reloadConfigErr == nil {
+							log.Infof("Successfully reloaded configuration from: %q", cfgPath)
+							break
+						}
+						log.Errorf("failed to reload configuration. err: %+v", reloadConfigErr)
```

Comment:
Can you also say `will try again in 5 seconds` or something like that ?

Reply:
change it to

```
log.Errorf("failed to reload configuration. will retry again in 5 seconds. err: %+v", reloadConfigErr)
```